packages/gatsby/src/redux/reducers/nodes-by-type.ts

```diff
@@ -1,12 +1,21 @@
-const getNodesOfType = (node, state) => {
+import { IGatsbyNode, IGatsbyState, ActionsUnion } from "../types"
+
+const getNodesOfType = (
+  node: IGatsbyNode,
+  state: IGatsbyState["nodesByType"]
+): Map<string, IGatsbyNode> => {
   const { type } = node.internal
   if (!state.has(type)) {
     state.set(type, new Map())
   }
-  return state.get(type)
+  return state.get(type) as Map<string, IGatsbyNode>
 }
 
-module.exports = (state = new Map(), action) => {
+// module.exports = (
```

Comment:
```suggestion
```

Reply:
Ow, I ended up forgetting that comment 😕